docs/CHANGELOG.asciidoc

```diff
@@ -29,6 +29,7 @@
 
 Improve and use periodic boundary condition for seasonal component modeling ({pull}84[#84])
 Improve robustness w.r.t. outliers of detection and initialisation of seasonal components ({pull}90[#90])
+Extend unit testing and make some improvements to deal with changes in seasonal components ({pull}91[#91])
```

Comment:
I don't think the release notes need to mention unit test changes - someone trying to find out what's new in ML isn't going to care.

Reply:
Fair enough, I'll update.